toolkits/springer/packages/springer-listing/scss/50-components/_listing.scss

```diff
@@ -1,80 +1,96 @@
-.c-listing {
+.c-listings {
```

Comment:
I think this needs to be `c-listing` in order to adhere to normal BEM convention. Or alternatively everything else needs to be `c-listings__`. As the BEM elements and modifiers are extensions of the block, so they should all share the same block value so people can easily see the relationship. What do you think?

Reply:
@benjclark Ah yes, sorry. I think option 2 is probably best.